src/hooks/fetchUtils.test.ts

```diff
@@ -0,0 +1,229 @@
+import { NODE_MAJOR_VERSION } from '../../testUtils/nodeVersion';
+import { FetchInstrumentation } from './fetch';
+
+// Conditionally skip the test suite
+if (NODE_MAJOR_VERSION < 18) {
+  describe.skip('fetchUtils.skip', () => {
+    it('should not run on Node.js versions less than 18', () => {
+      // test cases here
```

Comment:
I think we can remove this, the title says it all
```suggestion
      // test cases here
```

Reply:
Done